fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalHashJoin.java

```diff
@@ -38,9 +40,10 @@
         RIGHT_CHILD_TYPE extends Plan>
         extends AbstractPhysicalJoin<LEFT_CHILD_TYPE, RIGHT_CHILD_TYPE> {
 
-    public PhysicalHashJoin(JoinType joinType, Optional<Expression> condition, LogicalProperties logicalProperties,
+    public PhysicalHashJoin(JoinType joinType, List<Expression> hashJoinConjuncts,
+            Optional<Expression> condition, LogicalProperties logicalProperties,
```

Comment:
change `Optional<Expression> condition` to `List<Expression> nonhashJoinConjuncts`